### PR TITLE
Site reports

### PR DESCRIPTION
**Pull request by nikosdion**
Close gh-220

### CODE REVIEW

File: src/Model/Reports.php [hunk 1]
```diff
@@ -0,0 +1,869 @@
...
+		return $item;
+	}
+
+	/**
+	 * Returns a new object for an Extension Update Found event
+	 *
+	 * @param   int          $site_id        The side ID
+	 * @param   string       $extensionKey   The extension key, e.g. com_example
+	 * @param   string       $extensionName  The human-readable extension name, e.g. "An Example"
+	 * @param   string|null  $oldVersion     The old version of the CMS
```
Comment (brianteeman) [inline src/Model/Reports.php:138]: shouldn't this (and similar) be "the old versi9on of the extension"

Comment (nikosdion) [reply src/Model/Reports.php:138]: Yup. Wanna make a PR for it?

Comment (brianteeman) [reply src/Model/Reports.php:138]: ok - will do when I get back from the hsp[ital

Comment (nikosdion) [reply src/Model/Reports.php:138]: Thank you — and I hope it all goes well for you and your family.

**Pull request merged**